docs/operator-guides/security.md

```diff
@@ -1,46 +1,47 @@
 
 An Airbyte deployment stores the following data:
 
 ### Technical Logs
 
-Technical logs are stored for troubleshooting purposes and may contain sensitive data based on the connection’s `state` data. If your connection is set to an Incremental sync mode, users choose which column is the cursor for their connection. While we strongly recommend a timestamp like an `updated_at` column, users can choose any column they want to be the cursor. 
+Technical logs are stored for troubleshooting purposes and may contain sensitive data based on the connection’s `state` data. If your connection is set to an Incremental sync mode, users choose which column is the cursor for their connection. While we strongly recommend a timestamp like an `updated_at` column, users can choose any column they want to be the cursor.
 
 ### Configuration Metadata
 
-Airbyte retains configuration details and data points such as table and column names for each integration. 
+Airbyte retains configuration details and data points such as table and column names for each integration.
 
 ### Sensitive Data​
 
 As Airbyte is not aware of the data being transferred, users are required to follow the [Terms of Services](https://airbyte.com/terms) and are ultimately responsible for ensuring their data transfer is compliant with their jurisdiction.
 
 For more information, see [Airbyte’s Privacy Policy](https://airbyte.com/privacy-policy)
 
-## Securing Airbyte Open Source 
+## Securing Airbyte Open Source
 
 :::note
-Our security and reliability commitments are only applicable to Airbyte Cloud. Airbyte Open Source security and reliability depend on your development and production setups. 
+Our security and reliability commitments are only applicable to Airbyte Cloud. Airbyte Open Source security and reliability depend on your development and production setups.
 :::
 
 ### Network Security
 
-Deploy Airbyte Open Source in a private network or use a firewall to filter which IP addresses are allowed to access your host. Airbyte Open Source currently does not include any user management or role-based access controls (RBAC) to prevent unauthorized access to the API or UI. Controlling who has access to the hardware and network your Airbyte deployment runs on is your responsibility.
+Deploy Airbyte Open Source in a private network or use a firewall to filter which IP addresses are allowed to access your host. Airbyte Open Source currently does not include any user management or role-based access controls (RBAC) to prevent unauthorized access to the API or UI, only a HTTP basic auth password. Controlling who has access to the hardware and network your Airbyte deployment runs on is your responsibility.
```

Comment:
I think this section talks about access management and not authentication. I'm gonna say we don't need to add the password content here.

Reply:
Reverted in e537bdd6e25461026d09f648baf5e060701775a7